09_bg_change/index.js

```diff
@@ -0,0 +1,15 @@
+const btnChange = document.getElementById("change");
+const btnReset = document.getElementById("reset");
```

Comment:
There are multiple ways to target elements. It can be class or id.

id only returns one element (because we can only have 1 id in the page)
class can return multiple elements (in an array).

So, in this case, because you have only one element, id is just fine.

Personally I prefer to use classname instead of id. I rarely use ids, and when I use them is for something related with A11Y (Accessibility), not JS.

When using class, I use it like this:
```
// all classes used by JS are prepended by `.js-` so that it's clear they are meant to be used by JS, and not for styling purposes.

// get all elements with .js-btnChange
const btnsChange = document.getElementsByClassname("js-btnChange");
// get only the first element.
const btnChange = btnsChange[0]

// this is the same as 
const btnChange = document.getElementsByClassname("js-btnChange")[0];
```